src/main/java/edu/utdallas/davisbase/representation/SelectCommandRepresentation.java

```diff
@@ -0,0 +1,38 @@
+package edu.utdallas.davisbase.representation;
+
+import java.util.List;
+
+public class SelectCommandRepresentation implements CommandRepresentation {
```

Comment:
This class is missing the following getter methods:

- `public String getTable()`
- `public List<String> getColumn()`
- `public List<Expression> getValues()` (whether this is a catch-all `Expression` or our own `ValueRepresentation` class that permits only literal value expressions is up to you)

Also, shouldn't the class `SelectCommandRepresentation` have a `WhereExpression` field?